File: src/main/java/tw/com/box/BoxTester.java
package tw.com.box;

import java.util.Scanner;

public class BoxTester {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Please enter box length:");
        float length = scanner.nextFloat();
        System.out.print("Please enter box width:");
        float width = scanner.nextFloat();
        System.out.print("Please enter box height:");
        int height = scanner.nextInt();

        Box3 box3 = new Box3(23,14,13);
        Box5 box5 = new Box5(39.5f,27.5f,23);
        if (box3.validate(length,width,height)) {
            System.out.println("Box3");
        }
        else if (box5.validate(length,width,height)) {
            System.out.println("Box5");
        }
        else {
            System.out.println("Box out");
        }
    }
}

File: src/main/kotlin/tw/com/boxkotlin/BoxTester.kt
package tw.com.boxkotlin

import java.util.*

fun main(args: Array<String>) {
    val scanner = Scanner(System.`in`)
    print("Please enter box length:")
    val length = scanner.nextFloat()
    print("Please enter box width:")
    val width = scanner.nextFloat()
    print("Please enter box height:")
    val height = scanner.nextInt()

    val box3 = Box3(23f,14f,13)
    val box5 = Box5(39.5f,27.5f,23)
    if (box3.validate(length,width,height)) {
        println("Box3")
    }
    else if (box5.validate(length,width,height)) {
        println("Box5")
    }
    else {
        println("Box Out...")
    }
}